src/main/java/com/codingnomads/betty/data/models/Tweet.java

```diff
@@ -13,10 +13,12 @@
     private Long id;
     @Size(max = 15)
     private String language;
-    @NotBlank
+    @NotBlank()
+    @Size(max = 512)
```

Comment:
We must fix the limitation of characters

Reply:
Hey alp. I will address in my next branch to make sure Jordan can rebase from Master. Thank you!